.env

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_SUPABASE_URL=https://uyrlqjpqidyilpcpdiqb.supabase.co
```

Comment:
You should add this to the gitignore because we don't want the sensitive data in the repo! This is why we don't directly put these values in our code as variables.